### ISSUE TITLE
Library missing

### ISSUE THREAD

**Issue by onerfusion** [himphen/TestYourAndroid#1]
you have ActionBarSherlock
And google services library missing in ur repo

**Comment by himphen** [comment]
Yes, but there are two library projects and I don't know how to include them into this repo.
